src/core2h.rkt

```diff
@@ -0,0 +1,28 @@
+#lang racket
+
+(require "imperative.rkt" "core2c.rkt")
+(provide core->h)
+
+;; ignore body
+(define-expr-visitor imperative-visitor h-visitor
+  [(visit-if vtor cond ift iff #:ctx ctx) (values "" ctx)]
+  [(visit-let_ vtor let_ vars vals body #:ctx ctx) (values "" ctx)]
+  [(visit-while_ vtor while_ cond vars inits updates body #:ctx ctx) (values "" ctx)]
+  [(visit-for_ vtor for_ vars vals accums inits updates body #:ctx ctx) (values "" ctx)]
+  [(visit-tensor vtor vars vals body #:ctx ctx) (values "" ctx)]
+  [(visit-tensor* vtor vars vals accums inits updates body #:ctx ctx) (values "" ctx)]
+  [(visit-! vtor props body #:ctx ctx) (values "" ctx)]
+  [(visit-op_ vtor op_ args body #:ctx ctx) (values "" ctx)]
+  [(visit-terminal_ vtor x #:ctx ctx) (values "" ctx)])
```

Comment:
I assume these will never be called? Because we never visit the program body? In that case perhaps better leave them as the default visitor, which throws errors?

Reply:
Fixed. Added a `no-body` flag to the imperative compiler to avoid even descending into the body.